x-pack/plugins/security_solution/server/lib/detection_engine/scripts/roles_users/detections_admin/delete_detections_user.sh

```diff
@@ -0,0 +1,3 @@
+curl -v -H 'Content-Type: application/json' -H 'kbn-xsrf: 123'\
+ -u ${ELASTICSEARCH_USERNAME}:${ELASTICSEARCH_PASSWORD} \
+-XDELETE ${ELASTICSEARCH_URL}/_security/user/detections_admin
```

Comment:
Looks like both [Detection Engine](https://github.com/elastic/kibana/blob/1216b0f7cd2c0697393547432bee343ed33b96b5/x-pack/plugins/security_solution/server/lib/detection_engine/scripts/find_saved_object.sh#L14-L13) and [Case](https://github.com/elastic/kibana/blob/4fc89aeb0debe44632848c230f7a732aae6d5ff7/x-pack/plugins/case/server/scripts/patch_cases.sh#L18) scripts include the copyright header, so I think we'll want it for all these shell scripts as well.